lib/matplotlib/figure.py

```diff
@@ -256,23 +256,20 @@ def _update_this(self, s, val):
 class Figure(Artist):
 
     """
-    The Figure instance supports callbacks through a *callbacks*
-    attribute which is a :class:`matplotlib.cbook.CallbackRegistry`
-    instance.  The events you can connect to are 'dpi_changed', and
-    the callback will be called with ``func(fig)`` where fig is the
-    :class:`Figure` instance.
+    The Figure instance supports callbacks through a *callbacks* attribute
+    which is a `~.CallbackRegistry` instance.  The events you can connect to
```

Comment:
This resolves correctly?

Reply:
yes https://929-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/_as_gen/matplotlib.figure.Figure.html#matplotlib.figure.Figure

why would it not?

Reply:
Because sphinx's reference resolution system is a bit magical to me and I always resort to putting in the full path.

Reply:
pretty sure it warns (and thus the doc build fails :-)) when it cannot resolve properly.